### DISCUSSION TITLE
access token expires after I change access_token time from 1h to 720h

### DISCUSSION THREAD

**Discussion by amitpawar832ap** [ory/hydra#2960] [Q&A] [answered]
We have set the value of TTL_ACCESS_TOKEN to 720h, the token is still however expiring every 1h. 
The parameters we have listed below.

ttl:
        access_token: 720h
        refresh_token: 720h
        login_consent_request: 720h
        id_token: 720h
        auth_code: 720h

Hydra returns value for auth_time and exp time is,
auth_time: 1643363443
exp: 1643367081

Diff exp time - auth_time = 3600 sec (1h).

Any help would be appreciated.  This is an area of Hydra we seem to not have as much knowledge as we need to resolve.

**Comment by vinckr** [comment]
Are you sure your config is being loaded correctly? 
Can you share some info on how you are running Hydra, possibly full configuration?
If you have a reproducible case that you can put on GitHub that would be very helpful as well.

**Comment by amitpawar832ap** [comment, marked as answer]
Hi Vinckr,
Thanks for the reply!

Sorry for the delay but we are not able to resolve the issue still.  We have already set the id_token to 720h.
Could you please see the bellow configuration that we did?

![hydra](https://user-images.githubusercontent.com/38393673/171020389-db1e3b0e-2095-42be-9cf5-131412674349.png)

**Reply by vinckr** [reply]
Hey @amitpawar832ap , 
apologies for the late follow up,
just checking in if your problem still persists. 
I will mark this as answered for now, but feel free to come back to the discussion any time.
Thanks a ton :bee:
